client/cypress/integration/query/parameter_spec.js

```diff
@@ -142,7 +154,7 @@ describe('Parameter', () => {
         .click();
 
       cy.getByTestId('DynamicTable')
-        .should('contain', '15/01/19');
+        .should('contain', moment().format('15/MM/YY'));
```

Comment:
Can't import moment unless it's preinstalled (I assume) but you can use `Cypress.moment` which is a built in util.

Reply:
ya learn somethin' every day